functions/go/Makefile

```diff
@@ -37,6 +37,7 @@ FUNCTIONS := \
 	set-labels \
 	set-namespace \
 	set-project-id \
+	source-gcloud-config \
```

Comment:
ah, this is just the directory name. it can be called anything in kpt.

Reply:
I saw both `generate-gcloud-config` and `source-gcloud-config` are used in your doc. I picked source because I don't want this to be surfaced as a normal generator. Unlike other curated KRM functions, it 1.)does not manipulate the  input resources and 2.) it calls executable `gcloud` which is abnormal. It serves more as a pure (functionConfig) source provider.